pkg/scheduler/apis/config/v1beta3/default_plugins_test.go

```diff
@@ -451,6 +359,132 @@ func TestMergePlugins(t *testing.T) {
+					Enabled: []v1beta3.Plugin{
+						{Name: "CustomPlugin"},
+					},
+				},
+			},
+			defaultPlugins: &v1beta3.Plugins{
+				MultiPoint: v1beta3.PluginSet{
+					Enabled: []v1beta3.Plugin{
+						{Name: "DefaultPlugin1"},
+						{Name: "DefaultPlugin2"},
+					},
+				},
+			},
+			expectedPlugins: &v1beta3.Plugins{
+				MultiPoint: v1beta3.PluginSet{
+					Enabled: []v1beta3.Plugin{
+						{Name: "DefaultPlugin1"},
+						{Name: "DefaultPlugin2"},
+						{Name: "CustomPlugin"},
+					},
+				},
+			},
+		},
+		{
+			name: "Append disabled Multipoint plugins",
+			customPlugins: &v1beta3.Plugins{
+				MultiPoint: v1beta3.PluginSet{
+					Enabled: []v1beta3.Plugin{
+						{Name: "CustomPlugin"},
+						{Name: "CustomPlugin2"},
+					},
+					Disabled: []v1beta3.Plugin{
+						{Name: "DefaultPlugin2"},
+					},
+				},
+				Score: v1beta3.PluginSet{
+					Disabled: []v1beta3.Plugin{
+						{Name: "CustomPlugin2"},
+					},
+				},
+			},
+			defaultPlugins: &v1beta3.Plugins{
+				MultiPoint: v1beta3.PluginSet{
+					Enabled: []v1beta3.Plugin{
+						{Name: "DefaultPlugin1"},
+						{Name: "DefaultPlugin2"},
+					},
+				},
+			},
+			expectedPlugins: &v1beta3.Plugins{
+				MultiPoint: v1beta3.PluginSet{
+					Enabled: []v1beta3.Plugin{
+						{Name: "DefaultPlugin1"},
+						{Name: "CustomPlugin"},
+						{Name: "CustomPlugin2"},
+					},
+					Disabled: []v1beta3.Plugin{
+						{Name: "DefaultPlugin2"},
+					},
+				},
+				Score: v1beta3.PluginSet{
+					Disabled: []v1beta3.Plugin{
+						{Name: "CustomPlugin2"},
+					},
+				},
+			},
+		},
+		{
+			name: "override default MultiPoint plugins with custom value",
+			customPlugins: &v1beta3.Plugins{
+				MultiPoint: v1beta3.PluginSet{
+					Enabled: []v1beta3.Plugin{
+						{Name: "DefaultPlugin", Weight: pointer.Int32(5)},
+					},
+				},
+			},
+			defaultPlugins: &v1beta3.Plugins{
+				MultiPoint: v1beta3.PluginSet{
+					Enabled: []v1beta3.Plugin{
+						{Name: "DefaultPlugin"},
+					},
+				},
+			},
+			expectedPlugins: &v1beta3.Plugins{
+				MultiPoint: v1beta3.PluginSet{
+					Enabled: []v1beta3.Plugin{
+						{Name: "DefaultPlugin", Weight: pointer.Int32(5)},
+					},
+				},
+			},
+		},
+		{
+			name: "disabled MultiPoint plugin in default set",
+			defaultPlugins: &v1beta3.Plugins{
+				MultiPoint: v1beta3.PluginSet{
+					Enabled: []v1beta3.Plugin{
+						{Name: "DefaultPlugin"},
+					},
+					Disabled: []v1beta3.Plugin{
+						{Name: "DefaultPlugin2"},
+					},
+				},
+			},
+			customPlugins: &v1beta3.Plugins{
+				MultiPoint: v1beta3.PluginSet{
+					Enabled: []v1beta3.Plugin{
+						{Name: "CustomPlugin"},
+					},
+				},
+			},
+			expectedPlugins: &v1beta3.Plugins{
+				MultiPoint: v1beta3.PluginSet{
+					Enabled: []v1beta3.Plugin{
+						{Name: "DefaultPlugin"},
+						{Name: "CustomPlugin"},
+					},
+					Disabled: []v1beta3.Plugin{
+						{Name: "DefaultPlugin2"},
+					},
+				},
+			},
+		},
 	}
 
 	for _, test := range tests {
```

Comment:
(this is pre-existing)

Could you help update L492~L493 to:

```go
			gotPlugins := mergePlugins(test.defaultPlugins, test.customPlugins)
			if d := cmp.Diff(test.expectedPlugins, gotPlugins); d != "" {
				t.Fatalf("plugins mismatch (-want +got):\n%s", d)
			}
```

(setting the plugins back to subtest looks weird)

Reply:
Done